vmaas/cache.go

```diff
@@ -111,14 +111,6 @@ func ShouldReload(c *Cache, latestDumpEndpoint string) bool {
 	return false
 }
 
-func (c *Cache) errataIDs2Names(eids []int) []string {
```

Comment:
keep these functions and call ApplyMap inside of it. In my opinion the readability is better compared to using ApplyMap directly and it also can be better tested